hackathon_site/dashboard/frontend/src/slices/event/teamSlice.ts

```diff
@@ -91,6 +216,16 @@ export const isLoadingSelector = createSelector(
     (teamSlice) => teamSlice.isLoading
 );
 
```

Comment:
You should also create selectors for the error state. Because we're leaving all the error & constraint handling to the backend, you'll need a clearer way to display errors other than the snackbar.
Consider adding an error box in the modal if an error occurs:
<img src="https://user-images.githubusercontent.com/20618460/178184267-94ffa9eb-b99e-4648-a8bf-399e0da5ab90.png" width="200"/>

Reply:
Currently, we use a notification box to display the error msg. You would like me to make an error box in the modal, and show the error in the box, instead of the notification box, is that correct?

<img width="636" alt="CleanShot 2022-08-21 at 17 01 47@2x" src="https://user-images.githubusercontent.com/36619969/185810824-c39d9c96-91f4-4b01-b817-8c8628f5f5e5.png">

Reply:
Since you want to catch errors in the frontend before they occur, we don't need the extra error box the snackbar is fine as is.

Reply:
So just to confirm that I don't need to make any changes to my current code right now right?

Reply:
yeah no changes needed